### DISCUSSION TITLE
Contact Information

### DISCUSSION THREAD

**Discussion by SonoranBrian** [TGRHavoc/live_map#60] [Q&A]
Havoc,

I'm looking to get in contract to hire you for a small project, I've been unable to find a proper contact source for you on your website.

Please reach out to me anytime,
Email: brian@sonoransoftware.com
Discord: Brian S#7602

**Comment by TGRHavoc** [comment]
Sorry, not been at my computer for a while. Sorry about the contact information, just managed to update it.
Email: jordan@tgrhavoc.co.uk

Ill try and get back to you as soon as possible but, I can't promise anything at the moment (life is moving a bit fast)
